.aws/src/main.ts

```diff
@@ -83,10 +83,20 @@ class SnowplowSharedConsumerStack extends TerraformStack {
       config.eventBridge.prospectEventTopic
     );
 
+    // Consumer Queue should be able to listen to shareable-list and shareable-list-item events from shareable-lists-api.
+    const shareableListsApiEventTopicArn = `arn:aws:sns:${region.name}:${caller.accountId}:${config.eventBridge.prefix}-${config.environment}-${config.eventBridge.shareableListsApiEventTopic}`;
```

Comment:
no need to change, but it seems like this prefix could be pulled out and reused `arn:aws:sns:${region.name}:${caller.accountId}:${config.eventBridge.prefix}-${config.environment}-`